tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -51,8 +51,8 @@ Test#1 { "0" : { "$date" : 1416445411987 } }
 string(37) "{ "0" : { "$date" : 1416445411987 } }"
 string(37) "{ "0" : { "$date" : 1416445411987 } }"
 bool(true)
-Test#2 { }
-string(3) "{ }"
-string(3) "{ }"
+Test#2 { "0" : {  } }
+string(14) "{ "0" : {  } }"
+string(14) "{ "0" : {  } }"
```

Comment:
This doesn't make a whole lot of sense to me.
```
php -r '$d = new DateTime("now"); var_dump($d);'                                                                                                                                                                     <<<
object(DateTime)#1 (3) {
  ["date"]=>
  string(26) "2015-09-09 16:13:49.000000"
  ["timezone_type"]=>
  int(3)
  ["timezone"]=>
  string(19) "America/Los_Angeles"
}
```

Why wouldn't at least the public properties be serialized?